libaleth-interpreter/VMCalls.cpp

```diff
@@ -27,53 +27,49 @@ void VM::copyDataToMemory(bytesConstRef _data, intx::uint256*_sp)
 
 void VM::throwOutOfGas()
 {
-    BOOST_THROW_EXCEPTION(OutOfGas());
```

Comment:
Can we remove the definitions for these boost exceptions that you no longer throw, or are they still thrown in LegacyVM?